dali/dafilesrv/dafilesrv.cpp

```diff
@@ -432,11 +439,28 @@ int main(int argc,char **argv)
                 throttleSlowCPULimit = dafileSrvInstance->getPropInt("@throttleSlowCPULimit", throttleSlowCPULimit);
                 throttleSlowQueueLimit = dafileSrvInstance->getPropInt("@throttleSlowQueueLimit", throttleSlowQueueLimit);
 
-                authorizedOnly = dafileSrvInstance->getPropBool("@authorizedOnly", authorizedOnly);
+                const char *_rowServiceConfiguration = dafileSrvInstance->queryProp("@rowServiceConfiguration");
+                if (!isEmptyString(_rowServiceConfiguration))
+                    rowServiceConfiguration = _rowServiceConfiguration;
+                rowServicePort = dafileSrvInstance->getPropInt("@rowServicePort", rowServicePort);
             }
         }
         keyPairInfo = env->queryPropTree("EnvSettings/Keys");
     }
+    enum RowServiceCfg rowServiceCfg = rs_off;
+    if (rowServicePort)
+    {
+        if (streq("on", rowServiceConfiguration))
+            rowServiceCfg = rs_on;
+        else if (streq("both", rowServiceConfiguration))
+            rowServiceCfg = rs_both;
+#ifdef _USE_OPENSSL
+        else if (streq("onssl", rowServiceConfiguration))
+            rowServiceCfg = rs_onssl;
+        else if (streq("bothssl", rowServiceConfiguration))
+            rowServiceCfg = rs_bothssl;
+#endif
```

Comment:
What if config has rs_onssl but platform is built without _USE_OPENSSL ?

Reply:
it turns if off if !_USE_OPENSSL